### DISCUSSION TITLE
Add a tutorial with UL computation

### DISCUSSION THREAD

**Discussion by cosimoNigro** [gammapy/gammapy#4260] [Ideas]
I think it would be nice to have a notebook showing how to compute UL on a weak or non-detected source.
This is a science case as much as estimating a spectrum or a light curve.

In particular, the tutorial could show how to compute ULs for different confidence levels, or how to change the threshold statistics to be used to define them.

**Comment by registerrier** [comment]
Thanks @cosimoNigro . This is a good idea. Could this be part of a source detection tutorial? e.g. at the end of this [tutorial](https://docs.gammapy.org/1.0.1/tutorials/analysis-2d/detect.html)?

What would you show in a stand alone tutorial?
- how to use the `Fit.confidence` and `Fit.stat_profile` to explore the `amplitude` parameter UL?
- or use a `FluxEstimator` to do so?

**Reply by adonath** [reply]
Note that the `MapEstimator` also compute ULs, so for simple cases one would just read it off the UL map.

**Comment by Astro-Kirsty** [comment]
We know have two tutorials relating to this:
-  [computing flux upper limits for a non-detected source](https://docs.gammapy.org/dev/tutorials/analysis-3d/non_detected_source.html)
- [dealing with upper limits on parameters](https://docs.gammapy.org/dev/tutorials/details/parameter_ul.html)
